2.train-VAE/scripts/1.optimize_hyperparameters.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import sys
+import pathlib
+import numpy as np
+import pandas as pd
+import hiplot
+from optuna.visualization import plot_param_importances
```

Comment:
Also seems like some of these imports are unused? If so, consider removing them.

Reply:
Removed